### PR TITLE
add grpc-http-proxy chart

### PR DESCRIPTION
**Pull request by vbanthia-zz**
#### What this PR does / why we need it:
Add [grpc-http-proxy](https://github.com/mercari/grpc-http-proxy) chart

#### Special notes for your reviewer:
N/A

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by vbanthia** [comment]
@davidkarlsen  Could you look this PR.

**Comment by davidkarlsen** [comment]
@vbanthia Please aim straight for the stable repo with a 1.0.0 version

File: incubator/grpc-http-proxy/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "grpc-http-proxy.fullname" .  }}
+  labels: {{ include "grpc-http-proxy.labels" . | indent 4 }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```
Comment (davidkarlsen) [inline incubator/grpc-http-proxy/templates/deployment.yaml:6]: please add labels like described here: https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md

Comment (davidkarlsen) [reply incubator/grpc-http-proxy/templates/deployment.yaml:6]: please add rbac as described here: https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Pull request closed without merging**